### PR TITLE
slight change in parameters and ensure correct cropping

### PR DESCRIPTION
**Pull request by jdammers**
Slight changes in parameters. Usage of hp-filter prevents from false positives. Ensure correct cropping. Added more comments.

### CODE REVIEW

File: examples/decompose/do_MLICA.py [hunk 1]
```diff
@@ -39,76 +41,97 @@
...
+picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=False,
+                       stim=False, exclude='bads')
+
+# ----------------------------------------------
 # noise reducer
+# ----------------------------------------------
 raw_nr = noise_reducer(raw_fname, reflp=5., return_raw=True)
-
 raw_nr = noise_reducer(raw_fname, raw=raw_nr, refhp=0.1, noiseref=['RFG ...'],
                        return_raw=True)
```
Comment (pravsripad) [inline examples/decompose/do_MLICA.py]: We can also include a ```raw.close()  # close previous raw instance```

**Comment by jdammers** [comment]
Yes, only the file examples/decompose/do_MLICA.py has been modified.
I think the noise reducer part could be removed as in this example I load the noise reducer file already

File: examples/decompose/do_MLICA.py [hunk 2]
```diff
@@ -142,10 +164,10 @@
 raw_unfiltered_chop_clean = ica_unfiltered_chop.apply(raw_chop, exclude=ica.exclude, n_pca_components=None)
 
 # create copy of original data since apply_ica_replace_mean_std changes the input data in place (raw and ica)
-raw_copy = raw.copy().crop(tmin=tmin*4./1000, tmax=tmax*4./1000)
+raw_copy = raw.copy().crop(tmin=tmin, tmax=tmax)
```
Comment (pravsripad) [inline examples/decompose/do_MLICA.py]: This line can also be removed raw_chop can be used in the call to plot below.

**Comment by jdammers** [comment]
OK, I'll do the changes.

**Comment by pravsripad** [comment]
>  I think the noise reducer part could be removed as in this example I load the noise reducer file already

Yes.

**Comment by jdammers** [comment]
The notches could be removed as well, correct?

File: examples/decompose/do_MLICA.py [hunk 3]
```diff
@@ -39,76 +41,97 @@
...
-
-raw_filtered = raw.copy().filter(0., 45., picks=picks, filter_length='auto',
+# ----------------------------------------------
+# filtering and down sampling
+# ----------------------------------------------
+# filter prior to ICA
+raw_filtered = raw.copy().filter(flow_raw, flow_high, picks=picks, filter_length='auto',
                                  l_trans_bandwidth='auto', h_trans_bandwidth='auto',
-                                 n_jobs=njobs, method='fir', phase='zero',
+                                 n_jobs='cuda', method='fir', phase='zero',
```
Comment (pravsripad) [inline examples/decompose/do_MLICA.py]: Also, maybe its best to set n_jobs=njobs everywhere, and change the default to 'cuda' in line 32 above. Easier for testing on cpu only systems as well.

**Comment by pravsripad** [comment]
> The notches could be removed as well, correct?

I guess that the '109925_CAU01A_100715_0842_2_c,rfDC,t1,n_bcc,nr-raw.fif' file has been noise reduced in the standard manner with lp < 5hz and notches removed, best to double check once again.

**Comment by jdammers** [comment]
Looks like notches have been removed.
image

**Comment by jdammers** [comment]
@pravsripad final changes are done (see Cosmets). If you are happy we can merge.

**Comment by pravsripad** [comment]
Many thanks @jdammers .

**Pull request merged**